src/main/resources/mediathek/file/about/credits.html

```diff
@@ -11,6 +11,7 @@
 <br/>
 <br/>
 <b>Autoren (GitHub):</b><br/>
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Replace this <b> tag by <strong>. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)